feat(ET-480): Add documents for cdcreader installation (#5917)
EnterpriseDB/docs: product_docs/docs/transporter/ba_preview/getting_started/installing/linux_x86_64/transporter_debian_11.mdx

---
navTitle: Debian 11
title: Installing EDB Transporter on Debian 11 x86_64
---

## Prerequisites

Before you begin the installation process:

- Set up the EDB repository.

  Setting up the repository is a one-time task. If you have already set up your repository, you don't need to perform this step.

  To determine if your repository exists, enter:

  `apt-cache search enterprisedb`

  If no output is generated, the repository isn't installed.

  To set up the EDB repository:

  1. Go to [EDB repositories](https://www.enterprisedb.com/repos-downloads).

  1. Select the button that provides access to the EDB repo.

  1. Select the platform and software that you want to download.

## Install the package

> Install the EDB Data Migration Service Reader or CDCReader:

Reviewer: Hi @Xander-run !
While reviewing this with a colleague, we decided to call the reader "EBD DMS Reader (packaged as `cdcreader`)", could we update this to: 
```suggestion
Install the EDB DMS Reader (packaged as `cdcreader`):
```
Applies to this and other install pages.
